### PR TITLE
feat: added version of `click` in `install_requires` (close #32)

### LINKED ISSUE #32: Bug: bright_green not introduced until Click 7.0
Currently, `setup.py` just has Click without a version as a requirement. However, the color `bright_green` wasn't introduced until Click 7.0 resulting in the following error with older versions.

```
Traceback (most recent call last):
  File "/home/neil/.local/bin/create-aio-app", line 11, in <module>
    sys.exit(main())
  File "/home/neil/.local/lib/python3.6/site-packages/create_aio_app/main.py", line 44, in main
    fg='bright_green',
  File "/usr/local/lib/python3.6/dist-packages/click/termui.py", line 374, in style
    raise TypeError('Unknown color %r' % fg)
TypeError: Unknown color 'bright_green'
```
The following commit introduced bright colors to `termui.py`: https://github.com/pallets/click/commit/4383f608ef80168cd3f344b814caec548834c57a

### PR DESCRIPTION
**Pull request by Arfey**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def read_version():
 
 install_requires = [
     'cookiecutter',
-    'click',
+    'click==7.0',
```
[the remark this answers is not in the document]
Comment (Arfey) [reply setup.py]: Now, it's not critical. But i agree that `click>=7.0` look better. I will correct that in future.

[the remark this answers is not in the document]
Comment (webknjaz) [reply setup.py]: It is *very* critical. What if I have same env requiring `click<7`?

[the remark this answers is not in the document]
Comment (webknjaz) [reply setup.py]: What about `>=7.0.1`?

[the remark this answers is not in the document]
Comment (Arfey) [reply setup.py]: So, our util will not work because https://github.com/aio-libs/create-aio-app/issues/32

[the remark this answers is not in the document]
Comment (Arfey) [reply setup.py]: i will correct that :)

[the remark this answers is not in the document]
Comment (Arfey) [reply setup.py]: https://github.com/aio-libs/create-aio-app/pull/59/files

**Pull request merged**